tutorials/scripting/c_sharp/index.rst

```diff
@@ -42,13 +42,23 @@ C# platform support
 -------------------
 
 As of Godot 4.2 projects written in C# support all desktop platforms (Windows, Linux,
-and macOS), as well as Android. Android support is currently experimental and has a few
-limitations.
+and macOS), as well as Android.
 
-- Projects targeting Android need to be created using .NET 7.0 or higher. This requires
-  modifying the `.csproj` file generated by Godot.
+Android support is currently experimental and has a few limitations.
+
+- Projects targeting Android need to target .NET 7.0 or higher. This requires
+  modifying the `.csproj` file generated by Godot for existing projects.
 
 - Only some Android architectures are supported: ``arm64`` and ``x64``.
 
-Currently, projects written in C# cannot be exported to iOS and web platforms. To use C#
-on those platforms, consider Godot 3 instead.
+iOS support is currently experimental and has a few limitations.
+
+- Projects targeting iOS need to target .NET 8.0 or higher. This requires
+  modifying the `.csproj` file generated by Godot for existing projects.
+
+- Only some iOS architectures are supported: ``arm64``.
```

Comment:
I'm not sure this is the only architecture supported, also didn't mention anything about the simulator but I assumed we support the same architecture.

Reply:
There aren't any architecture limitations for iOS, on the C# export side - it's arm64 for the device builds, and both x64 and arm64 for simulator builds.

The official template package doesn't include the simulator arm64 template, though, so users won't be able to create final packages for that specific combo out of the box. They can build the templates themselves and that would work.

Reply:
What about `arm32` and `x86_64`? Does iOS not support those architectures? For the simulator we can mention that we don't provide official templates for `arm64` (so it's only `x64` right?).

Reply:
No, iOS doesn't support anything other than arm64, not since iPhone 5. The simulator supports arm64 and x64 only because Apple still supports Intel macs, they haven't deprecated everything that's not silicon (yet).

Reply:
Yup, `x64` for the simulator!

Reply:
OK, I removed the part about iOS architectures and only added a note about the iOS simulator templates limitations, let me know if it looks good now.